app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntrySheetController.kt

```diff
@@ -456,6 +456,8 @@ class BloodPressureEntrySheetController @Inject constructor(
 
     val currentFacilityStream = loggedInUserStream
         .flatMap(facilityRepository::currentFacility)
+        .replay()
```

Comment:
Why was this added?

Reply:
Since the stream is being consumed at two different places, `facilityRepository::currentFacility` was being invoked twice. I added this after the characteristic test showed two invocations.

Since this is a behavior we won't be able to replicate in Mobius, I added `replay()` ran the tests, ran the app and ensured the behavior remained the same.

Reply:
Will a `share()` work instead of a `replay().refCount()`?

Reply:
`share()` had problems in one of the older migration. Hence avoiding it.